Makefile

```diff
@@ -413,6 +413,28 @@ testdata:
 go-mod-tidy:
 	find ./ -type f -name go.mod -execdir go mod tidy \;
 
+.PHONY: update-vmlinux
+update-vmlinux:
+	KERNEL_VERSION=$${KERNEL_VERSION:-6.15.5} && \
+	IMG=$${CI_KERNELS_REPO:-ghcr.io/inspektor-gadget/ci-kernels}:$${KERNEL_VERSION}-debug && \
```

Comment:
I don't see `ghcr.io/inspektor-gadget/ci-kernels:6.15.5-debug` in the registry. Or it needs to be pushed for this to work?

Reply:
Should CI_KERNELS_REPO be declared as a variable at the beginning of the makefile?

Reply:
Ok. I named the variable `CI_KERNELS_IMAGE` instead and I made it reuse `CONTAINER_REPO_NAMESPACE`. So it could be used in this way:
```
CONTAINER_REPO_NAMESPACE=ghcr.io/alban make update-vmlinux
```